tests/shopify_python/test_google_styleguide.py

```diff
@@ -198,3 +198,24 @@ def fnc():
         message = pylint.testutils.Message('complex-list-comp', node=bad_list_comp)
         with self.assertAddsMessages(message):
             self.walk(root)
+
+    def test_use_cond_exprs(self):
+        root_to_msg = astroid.builder.parse("""
+            if 1 < 2:
+                num = 1
+            else:
+                num = 2
+            """)
+        if_node = root_to_msg.body[0]
+        message = pylint.testutils.Message('cond-expr', node=if_node)
+        with self.assertAddsMessages(message):
+            self.walk(root_to_msg)
+
+        root_to_not_msg = astroid.builder.parse("""
```

Comment:
For brevity I recommended that he combine the two. We could have just the test above with a parsed expression containing a conditional that would pass and a conditional that would fail and expect only one message (and add comments to denote which is which).